github/Organization.py

```diff
@@ -18,6 +18,7 @@
 # Copyright 2018 Anton Nguyen <afnguyen85@gmail.com>                           #
 # Copyright 2018 sfdye <tsfdye@gmail.com>                                      #
 # Copyright 2018 Jacopo Notarstefano <jacopo.notarstefano@gmail.com>           #
+# Copyright 2018 bbi-yggy <yossarian@blackbirdinteractive.com>                 #
```

Comment:
What's the protocol on copyright notices? I don't mind credit, though tbh I'd rather just include my github username and not my email.

Reply:
There is actually a script (inside this repo) to generate all the credits, from git data. The maintainer is supposed to run the script from time to time to update. So I guess your email information will still be public anyway.